public/js/admin.js

```diff
@@ -4,6 +4,25 @@ var autocompleteTopics;
 $(function(){
     /* show the account table by default */
     displayQuestionTable();
+
+        $.ajax({
+            type: 'POST',
+            url: '/login',
+            data: {user:'ad', passwd: 'hi'},
```

Comment:
What is this whole block? And why are there hardcoded credentials?

Reply:
Lol I was testing Darshana stuff. I will remove it. Nice catch